### PR TITLE
OV-11: Header component

### PR DESCRIPTION
**Pull request by sergiy4**
This PR adds the Header component.
image
image

### CODE REVIEW

**Review by anton-otroshchenko — DISMISSED** [review]

File: frontend/src/app/app.tsx [hunk 1]
```diff
@@ -27,6 +31,7 @@ const App: React.FC = () => {
 
     return (
         <>
+            <Header />
```
Comment (o-nedashkivska) [inline frontend/src/app/app.tsx:34]: I guess, we should hide header when the path is /sign-in or /sign-up

Comment (sergiy4) [reply frontend/src/app/app.tsx:34]: Maybe, but I guess this functionality is for another task because I didn't have this in my task.

**Review by o-nedashkivska — DISMISSED** [review]

File: frontend/src/app/app.tsx (hunk 1 above)
Comment (nikita-remeslov) [reply frontend/src/app/app.tsx:34]: Header should be only on internal pages

Comment (nikita-remeslov) [reply frontend/src/app/app.tsx:34]: because you add header to the root app

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
check my comments

File: frontend/src/app/app.tsx (hunk 1 above)
Comment (sergiy4) [reply frontend/src/app/app.tsx:34]: Okay, I'll fix it

File: frontend/src/app/app.tsx [hunk 2]
```diff
@@ -27,6 +31,11 @@ const App: React.FC = () => {
 
     return (
         <>
+            {pathname === AppRoute.SIGN_UP ||
+            pathname === AppRoute.SIGN_IN ? null : (
+                <Header />
+            )}
+
```
Comment (anton-otroshchenko) [inline frontend/src/app/app.tsx:38]:
```suggestion
            {(pathname === AppRoute.SIGN_UP ||
            pathname === AppRoute.SIGN_IN) && (
                <Header />
            )}

```

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

**Review by o-nedashkivska — APPROVED** [review]

**Review by stefano-lacorazza — APPROVED** [review]

**Review by o-nedashkivska — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**